wrapper/src/main/java/software/amazon/jdbc/util/SqlMethodAnalyzer.java

```diff
@@ -84,16 +84,32 @@ public boolean doesCloseTransaction(final Connection conn, final String methodNa
     }
 
     final String statement = getFirstSqlStatement(String.valueOf(args[0]));
-    return isStatementClosingTransaction(statement);
+    return isTransactionBoundary(statement);
   }
 
-  public boolean isExecuteDml(final String methodName, final Object[] args) {
+  public boolean isTransactionBoundary(String statement) {
+    return statement.startsWith("COMMIT")
+        || statement.startsWith("ROLLBACK")
+        || statement.startsWith("END");
```

Comment:
Can we remove "END"? I can't remember why this was included originally, as far as I can tell it isn't a command used to close transactions

Reply:
Because it is in PostgreSQL

Reply:
https://www.postgresql.org/docs/current/sql-end.html